autovalet_husky/src/autovalet_husky/find_parking_spot.py

```diff
@@ -69,8 +70,12 @@ def pub_two_goals(self):
         self.goal1 = self.generate_parking_goal(self.tag_tf, pos1, rot1)
         while not parking:
             rospy.sleep(0.5)
+            # keep moving w/ goal gen until goal1 is in costmap
+            if self.dist_to_goal(self.goal1) > 4.0:
```

Comment:
should this number (4) also be read off parameter server?

Reply:
+1 I think these better be read off the server

Reply:
done